src/validation.cpp

```diff
@@ -631,17 +632,13 @@ static bool AcceptToMemoryPoolWorker(const CChainParams& chainparams, CTxMemPool
                 // first-seen mempool behavior should be checking all
                 // unconfirmed ancestors anyway; doing otherwise is hopelessly
                 // insecure.
+                // All transactions in mempool become replaceable after the timeout.
                 bool fReplacementOptOut = true;
                 if (fEnableReplacement)
                 {
```

Comment:
could you fix these brackets while here? ie same line?

Reply:
@PastaPastaPasta - PRs shouldn't 'fix' style for code they're not otherwise touching.